openshift/templates/api/api-dc-template.yaml

```diff
@@ -76,7 +76,7 @@ objects:
                 memory: 100Mi
             readinessProbe:
               httpGet:
-                path: /api/v1/health
```

Comment:
Are we not going to be versioning our API?

Reply:
By the BC Gov standards, unless we don't have v2 code available, we don't add v1 in our paths and keep it plain. Adam and I figured this out recently.

Reply:
👍🏼